CMakeLists.txt

```diff
@@ -130,8 +121,7 @@ function(spvtools_default_compile_options TARGET)
 
   if (${COMPILER_IS_LIKE_GNU})
     target_compile_options(${TARGET} PRIVATE
-      -std=c++11 -fno-exceptions -fno-rtti)
-    target_compile_options(${TARGET} PRIVATE
+      -fno-exceptions -fno-rtti
```

Comment:
Why was -std=c++11 removed?

Reply:
On [lines 27-29](https://github.com/KhronosGroup/SPIRV-Tools/pull/2784/files/7fed08aa3c24d6102b948e348fb749f7c55c26aa#diff-af3b638bc2a3e6c650974192a53c7291R27) we set the `CXX_STANDARD` already. `-std=c++11` is redundant here.